src/frontend/src/components/public/PIAIntakeResults/index.tsx

```diff
@@ -0,0 +1,105 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFileDownload } from '@fortawesome/free-solid-svg-icons';
+import { Link } from 'react-router-dom';
+import { IPPQResult } from '../../../types/interfaces/ppq-result.interface';
+import { API_ROUTES } from '../../../constant/apiRoutes';
+import {
+  FileDownload,
+  FileDownloadTypeEnum,
+} from '../../../utils/file-download.util';
+import Spinner from '../../common/Spinner';
+import { useState } from 'react';
+import Alert from '../../common/Alert';
+import Messages from './messages';
+import MDEditor from '@uiw/react-md-editor';
+
+interface IComponentProps {
+  result: IPPQResult;
+}
+
+const PIAIntakeResults = (props: IComponentProps) => {
+  const { result } = props;
+  const [isDownloading, setIsDownloading] = useState(false);
+  const [downloadError, setDownloadError] = useState('');
+
+  const handleDownload = async () => {
+    setDownloadError('');
+
+    if (!result?.id) {
+      console.error(
+        'Something went wrong. Result Id not available for download',
+      );
+      setDownloadError('Something went wrong. Please try again.');
+      return;
+    }
+
+    setIsDownloading(true);
+    try {
+      await FileDownload.download(
+        API_ROUTES.PPQ_RESULT_DOWNLOAD.replace(':id', `${result.id}`),
+        FileDownloadTypeEnum.PDF,
+      );
+    } catch (e) {
+      setDownloadError('Something went wrong. Please try again.');
+    } finally {
+      setIsDownloading(false);
+    }
+  };
+
+  const handleAlertClose = () => {
+    setDownloadError('');
+  };
+
+  return (
+    <div className="results results-wrapper pia-result">
```

Comment:
This will break based on the new css guidelines. I am certain this will not render exactly as per the UI/UX designs.